src/data/set/finite.lean

```diff
@@ -451,6 +451,20 @@ begin
     { exact ih c hcs hbc } }
 end
 
+section
+
+local attribute [instance, priority 1] classical.prop_decidable
+local attribute [instance, priority 0] set.decidable_mem_of_fintype
+
+lemma to_finset_card {α : Type*} [fintype α] (H : set α) :
```

Comment:
Should this be a simp-lemma? Which of the two sides is simp-normal-form?

Reply:
I don't know, which is why I didn't mark it as one. They have the same depth (because of the coercion on the rhs).